File: CooCooBase/Models/Core Data/Event+CoreDataProperties.m
//
//  Event+CoreDataProperties.m
//  
//
//  Created by omniwyse on 14/03/18.
//
//

#import "Event+CoreDataProperties.h"

@implementation Event (CoreDataProperties)

+ (NSFetchRequest<Event *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Event"];
}

@dynamic identifier;
@dynamic ticketid;
@dynamic clickedTime;
@dynamic amountRemaining;
@dynamic fare;
@dynamic type;
@dynamic walletContentUsageIdentifier;
@dynamic ticketActivationExpiryDate;
@end

File: Genfare/ManagedObjects/Event+CoreDataProperties.swift
//
//  Event+CoreDataProperties.swift
//  
//
//  Created by vishnu on 08/02/19.
//
//

import Foundation
import CoreData


extension Event {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Event> {
        return NSFetchRequest<Event>(entityName: "Event")
    }

    @NSManaged public var amountRemaining: NSNumber?
    @NSManaged public var clickedTime: NSNumber?
    @NSManaged public var fare: NSNumber?
    @NSManaged public var identifier: String?
    @NSManaged public var ticketActivationExpiryDate: String?
    @NSManaged public var ticketid: String?
    @NSManaged public var type: String?
    @NSManaged public var walletContentUsageIdentifier: String?

}
